rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/util/EvaluatorUtil.java

```diff
@@ -163,12 +164,16 @@ public static CompletableFuture<Evaluator> makeFutureEvaluator(ExecutorService e
                 eval.getConfiguration().setRascalJavaClassPathProperty(System.getProperty("rascal.compilerClasspath"));
                 eval.addClassLoader(RascalLanguageServer.class.getClassLoader());
                 eval.addClassLoader(IValue.class.getClassLoader());
+                ClassLoader cl = new SourceLocationClassLoader(pcfg.getClassloaders(), ShellEvaluatorFactory.class.getClassLoader());
```

Comment:
This requires an `append(pcfg.getBin())` to make sure classes in the current target folder are reachable as well.

Reply:
So `pcfg.getClassLoaders().append(pcfg.getBin())`  I believe. 

On the lib:///'less branchof rascal this will become `pcfg.getLibsAndTarget()`